app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("visitor is not vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
Whenever you are not sure about the incoming dict structure, prefer using `.get()` to retrieve values.